pkg/parser/keywords.go

```diff
@@ -247,6 +247,7 @@ var Keywords = []KeywordsType{
 	{"VARCHAR", true, "reserved"},
 	{"VARCHARACTER", true, "reserved"},
 	{"VARYING", true, "reserved"},
+	{"VECTOR", true, "reserved"},
```

Comment:
Differ from MySQL, this is changed to revered keyword?

Reply:
MySQL does not currently support vector keys: [link](https://dev.mysql.com/doc/refman/9.0/en/vector.html).
Indexing-related keywords such as `fulltext` and `unique` are reserved, so this is also done here. In addition, it is a little complicated to process without a reserved word.

Reply:
PM(@joey-yez ) comments that if it is used as the reserved keyword, there will be compatibility. So I will update it.